src/app/shell/shell.component.ts

```diff
@@ -22,12 +23,17 @@ export class ShellComponent implements OnInit, OnDestroy {
   constructor(private geolocationService: GeolocationService, private store: Store) {}
 
   ngOnInit(): void {
-    this.geolocationService.handleUserLocation((coords: Coords) => {
-      coords &&
-        this.geolocationService.getNearestByCoordinates(coords, (result: Codeficator) => {
-          this.geolocationService.confirmCity(result);
-        });
-    });
+    if (this.geolocationService.isCityInStorage()) {
+      this.geolocationService.confirmCity(JSON.parse(localStorage.getItem('cityConfirmation')), true);
+    } else {
```

Comment:
on init, if we don't location is forbidden and the city is not confirmed, then we should set Kyiv but not confirm it (the confirmation modal should be shown) 
![image](https://user-images.githubusercontent.com/27493959/196130183-8e42d920-1a1b-4c69-8b44-48ca886a70c1.png)